lib/instrument/transformer/dtx-encode.js

```diff
@@ -0,0 +1,32 @@
+import Stream from 'stream';
+import {DTXMessage} from '../headers';
+
+
+class DTXEncoder extends Stream.Transform {
+
+  constructor () {
+    super({ objectMode: true });
+  }
+
+  _transform (data, encoding, onData) {
+    this.push(this._encode(data), 'buffer');
+    onData();
+  }
+
+  _encode (data) {
+    const {sync, channelID, selector, auxiliaries, identifier} = data;
+    const dtx = new DTXMessage();
+    dtx._identifier = identifier;
```

Comment:
Methods/property names starting with underscore are considered private. Would it be possible to change the class constructor, so we could pass them as arguments upon instance creation?